RELEASE_NOTES

```diff
@@ -1,4 +1,35 @@
 
+###############################################################################
+#                                                                             #
+# Trilinos Release 15.0 Release Notes                                         #
+#                                                                             #
+###############################################################################
+
+CMake
+
+  - Change the default for `Trilinos_MUST_FIND_ALL_TPL_LIBS` from `OFF` to
+    `ON`, by setting `Trilinos_MUST_FIND_ALL_TPL_LIBS_DEFAULT` to `ON`. It was
+    turned off by default because it breaks backward compatibility but it also
+    causes problems for new users and new configurations.
+
+    Users that do not want this new behavior can set
+    `-D Trilinos_MUST_FIND_ALL_TPL_LIBS=OFF`, which is backward compatible.
+
+  - Change the default for `Trilinos_USE_GNUINSTALLDIRS` from `OFF` to `ON`,
+    by setting `Trilinos_USE_GNUINSTALLDIRS_DEFAULT` to `ON`, in the goal to
```

Comment:
```suggestion
    in the goal to
```

Again, you don't need to tell the user how the default was changed.  That is an implementation detail.